packages/supplementary-contracts/script/VestToken.s.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "forge-std/src/Script.sol";
+import "forge-std/src/console2.sol";
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import "../contracts/TokenUnlocking.sol";
+
+contract VestToken is Script {
+    using stdJson for string;
+
+    struct VestTokenRecipient {
+        bytes32 nameBytes; // Conversion from json "string" to bytes32 will take place in foundry,
+            // cannot use string here, as json parser cannot interpret string from json, everything
+            // is bytes-chunks. It is more of informational to script executor anyways.
+        address grantRecipientEOA;
+        address unlockingContractProxyAddress;
+        uint256 vestAmount;
+    }
+
+    struct VestTokenJsonData {
+        VestTokenRecipient[] tokenVests;
+    }
+
+    uint256 public OWNER_UNLCOKING_CONTRACTS_PK = vm.envUint("OWNER_UNLOCKING_PK"); // Who can call
+        // the vest() function
+    uint256 public OWNER_VAULT_PK = vm.envUint("OWNER_VAULT_PK"); // Who can call the approve()
```

Comment:
what are OWNER_UNLCOKING_CONTRACTS_PK and OWNER_VAULT_PK

Reply:
One PK (`OWNER_VAULT_PK`) is responsible for giving the `approve` on TKO token, so that `UnlockingContract` can `transferFrom`.
The other is the owner of the unlocking contract. These 2 can be the same, and if indeed the same, 1 can be removed.

Reply:
I see, how about name them:

DEPLOYER (msg.sender() and OWNER?